config/main.py

```diff
@@ -3149,6 +3149,45 @@ def update(args):
     except subprocess.CalledProcessError as e:
         sys.exit(e.returncode)
 
+#
+# 'chassis_modules' subgroup ('config platform chassis_modules ...')
+#
+@platform.group()
+@click.pass_context
+def chassis_modules (ctx):
```

Comment:
can you follow example here? https://github.com/Azure/sonic-utilities/blob/master/show/feature.py

move chassis_modules into its own file?

Reply:
Done.